taker/src/main.rs

```diff
@@ -340,6 +341,11 @@ async fn main() -> Result<()> {
         taker_peer_id: identities.libp2p.public().to_peer_id().to_string(),
     };
 
+    let environment = match option_env!("ITCHYSATS_ENV") {
+        Some(environment) => Environment::from_str_or_fallback(environment),
```

Comment:
🙃 Thanks! Maybe this name would be a little more natural.

```suggestion
        Some(environment) => Environment::from_str_or_unknown(environment),
```